package-lock.json

```diff
@@ -12700,12 +12700,13 @@
       "integrity": "sha512-YZo3K82SD7Riyi0E1EQPojLz7kpepnSQI9IyPbHHg1XXXevb5dJI7tpyN2ADxGcQbHG7vcyRHk0cbwqcQriUtg=="
     },
     "node_modules/sass": {
-      "version": "1.44.0",
-      "resolved": "https://registry.npmjs.org/sass/-/sass-1.44.0.tgz",
-      "integrity": "sha512-0hLREbHFXGQqls/K8X+koeP+ogFRPF4ZqetVB19b7Cst9Er8cOR0rc6RU7MaI4W1JmUShd1BPgPoeqmmgMMYFw==",
+      "version": "1.45.0",
```

Comment:
I don't think this PR updates dependancies so we should revert those package-lock.json changes. Other than that it's good to go.

Reply:
These are package lock changes that I believe were not versioned during the previous PR into `canary`, so I'm doing it now to make sure we're up to par.